packages/react-components/react-progress/src/components/ProgressBar/ProgressBar.test.tsx

```diff
@@ -18,7 +18,13 @@ describe('ProgressBar', () => {
       ],
     },
   });
-
+  const originalConsoleError = console.error;
+  beforeEach(() => {
```

Comment:
You can use `beforeAll`

```suggestion
  beforeAll(() => {
```